ios/Classes/SwiftWebviewCookieManagerPlugin.swift

```diff
@@ -93,6 +107,17 @@ public class SwiftWebviewCookieManagerPlugin: NSObject, FlutterPlugin {
         }
     }
     
+    public static func removeCookies(url: String?, result: @escaping FlutterResult) {
```

Comment:
Maybe it would be better to update clearCookies method with optional URL parameter?

Reply:
maybe